sdk/iOS/src/MSQueryResult.h

```diff
@@ -10,7 +10,7 @@
 ///@{
 
 /// The result from a successful read
-@property (nonatomic, strong, readonly)           NSArray *items;
+@property (nonatomic, strong, readonly, nonnull)           NSArray<NSDictionary *> *items;
```

Comment:
I'm not sure on items having to be non null, logically it probably should be, but I dont think the code base always gives an array depending on the server response.  Did you confirm this?